microsoft-edge/webview2/get-started/hololens2.md

```diff
@@ -0,0 +1,602 @@
+      public Button BackButton;
+      public Button GoButton;
+      public TMP_InputField URLField;
+
+      private void Start()
+      {
+         // Get the WebView component attached to the game object
+         var webViewComponent = gameObject.GetComponent<WebView>();
+         webViewComponent.GetWebViewWhenReady((IWebView webView) =>
+         {
+               // If the WebView supports browser history, enable the back button
+               if (webView is IWithBrowserHistory history)
+               {
+                  // Add an event listener for the back button to navigate back in history
+                  BackButton.onClick.AddListener(() => history.GoBack());
+
+                  // Update the back button's enabled state based on whether there's any history to go back to
+                  history.CanGoBackUpdated += CanGoBack;
+               }
+
+               // Add an event listener for the go button to load the URL entered in the input field
+               GoButton.onClick.AddListener(() => webView.Load(new Uri(URLField.text)));
+
+               // Subscribe to the Navigated event to update the URL input field whenever a navigation occurs
+               webView.Navigated += OnNavigated;
+
+               // Set the initial value of the URL input field to the current URL of the WebView
+               if (webView.Page != null)
+               {
+                  URLField.text = webView.Page.AbsoluteUri;
+               }
+         });
+      }
+
+      // Update the URL input field with the new path after navigation
+      private void OnNavigated(string path)
+      {
+         URLField.text = path;
+      }
+
+      // Enable or disable the back button based on whether there's any history to go back to
+      private void CanGoBack(bool value)
+      {
+         BackButton.enabled = value;
+      }
+   }
+   ```
+
+1. Modify the scene hierarchy as follows:
+   1. Under `MixedRealitySceneContent`, add a new input component (right-click > **UI** > **Input Field - TextMeshPro**). This automatically adds a parent `Canvas` for the component.
+   1. Under the new `Canvas`, add **two** new `Button` components (right-click > **UI** > **Button - TextMeshPro**).
+   1. Reorder the `WebView` component so that it is a child of the `Canvas` component:
+
+   ![Updated scene hierarchy with new components in Unity](./hololens2-images/unity_scene_hierarchy.png)
+
+1. Update the Canvas properties by selecting the Canvas in the **Hierarchy** pane, then making the following changes in the **Inspector** pane:
+   1. Change **Width** & **Height** to 600, 400 respectively.
+   1. Change **Scale** for X, Y, Z to 0.001, 0.001, 0.001:
+
+   ![Updated Canvas properties in Unity's Inspector](./hololens2-images/unity_canvas_inspector.png)
+
+1. Update the input field's properties by selecting it in the **Hierarchy** pane, then making the following changes in the **Inspector** pane:
+   1. Change **Name** to "AddressWell (TMP)"
```

Comment:
Which string is correct?  The text or png needs fix.
Text:   AddressWell (TMP)
.png:  AddressField (TMP)

gh rendered section:
https://github.com/michaelfarnsworth/edge-developer/blob/webview2-hololens-getting-started/microsoft-edge/webview2/get-started/hololens2.md#step-6---configure-webview2-settings-and-properties
find "AddressWell"

direct link to png:
https://github.com/michaelfarnsworth/edge-developer/raw/webview2-hololens-getting-started/microsoft-edge/webview2/get-started/hololens2-images/unity_addresswell_inspector.png

Reply:
Let's just go with AddressField (TMP).